vendor/engines/c2po/app/services/credit_card_account_builder.rb

```diff
@@ -0,0 +1,20 @@
+# Contains overrides for building a `CreditCardAccount` from params.
+# Dynamically called via the `AccountBuilder.for()` factory.
+class CreditCardAccountBuilder < AccountBuilder
+
+  # Override and append `build`.
+  def build
```

Comment:
What would you think of changing this to an `after_build` method that gets called in the parent. That way it doesn't need to call `super`.